spec/models/game_spec.rb

```diff
@@ -40,22 +40,22 @@
   describe Game, "scope" do
     it "Show games with white and black player present as closed" do
       game = FactoryGirl.create(:game, :full) 
-      expect(Game.available.map(&:id)[0]).to eq(nil)      
+      expect(Game.available.map(&:game_status)[0]).to eq(nil)

```

Comment:
Where is the list of possible game statuses? Shouldn't we define a constant somewhere?